azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,88 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps[
+        "iot hub state"
+    ] = """
+        type: group
+        short-summary: Manage the state of an IoT Hub.
+    """
+
+    helps[
+        "iot hub state export"
+    ] = """
+        type: command
+        short-summary: Export the state of an IoT Hub to a file.
+        long-summary: The exported state will include hub configurations, edge deployments, device identities and twins, and module
+                      identities and twins.
+        examples:
+        - name: Export the full state of the specified hub to the specified file.
```

Comment:
nit, is "full state" accurate?

Reply:
will replace to be all supported aspects